### PR TITLE
Add CreateOrUpdate as a new way of updating Users

### PR DESCRIPTION
**Pull request by eski009**
Currently there is no way to use the createOrUpdate API call.

### CODE REVIEW

**Comment by joseconsador** [comment]
@eski009 Thanks for this, there are some errors on the unit test though. If you could fix them on your end, we'd be happy to re-review this PR.

File: src/Zendesk/API/Resources/Core/Users.php [hunk 1]
```diff
@@ -39,6 +39,7 @@ class Users extends ResourceAbstract
         findMany as traitFindMany;
     }
     use UpdateMany;
+    use CreateOrUpdateMany;
```
Comment (miogalang) [inline src/Zendesk/API/Resources/Core/Users.php]: This seems to be throwing an error. I think a `use` is missing up top

**Comment by eski009** [comment]
@miogalang  - thanks. Have updated

**Comment by eski009** [comment]
@joseconsador - should be fixed.

File: src/Zendesk/API/Traits/Resource/CreateOrUpdateMany.php [hunk 2]
```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Zendesk\API\Traits\Resource;
+
+use Zendesk\API\Exceptions\RouteException;
+use Zendesk\API\Http;
+
+/**
+ * Allows resources to call a bulk show endpoint.
```
Comment (joseconsador) [inline src/Zendesk/API/Traits/Resource/CreateOrUpdateMany.php]: Needs updating.

**Comment by joseconsador** [comment]
:+1: after @eski009 adds the comment typo fix

**Comment by miogalang** [comment]
Some tests needed for this but that can be done by us. Escalating the writing of unit tests for this to Jira.

**Comment by miogalang** [comment]
:+1: thanks @eski009

**Pull request merged**